migrating_from_ocp_3_to_4/index.adoc

```diff
@@ -0,0 +1,82 @@
+== Upgrading MTC
+You xref:../migrating_from_ocp_3_to_4/upgrading-3-4.adoc#upgrading-3-4[upgrade the {mtc-full} ({mtc-short})] on {product-title} {product-version} by using OLM. You upgrade {mtc-short} on {product-title} 3 by reinstalling the legacy {mtc-full} Operator.
+
+[id="mtc-overview-mtc-checklists"]
+== Reviewing MTC checklists
+Before you migrate your application workloads with the Migration Toolkit for Containers (MTC), review the xref:../migrating_from_ocp_3_to_4/premigration-checklists-3-4.adoc#premigration-checklists-3-4[premigration checklists].
+
+[id="mtc-overview-migrate-mtc-applications"]
+== Migrating applications
+You can migrate your applications by using the MTC xref:../migrating_from_ocp_3_to_4/migrating-applications-3-4.adoc#migrating-applications-mtc-web-console_migrating-applications-3-4[web console] or xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#migrating-applications-cli_advanced-migration-options-3-4[the command line].
+
+[id="mtc-overview-advanced-migration-options"]
+== Advanced migration options
+You can automate your migrations and modify the MTC custom resources to improve performance of large-scale migrations by using the following options:
+
+* xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#migration-state-migration-cli_advanced-migration-options-3-4[Running a state migration]
+* xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#migration-hooks_advanced-migration-options-3-4[Creating migration hooks]
+* xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#migration-plan-options_advanced-migration-options-3-4[Editing, excluding, and mapping migrated resources]
+* xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#migration-controller-options_advanced-migration-options-3-4[Configuring the migration controller for large migrations]
+
+[id="mtc-overview-troubleshooting-mtc"]
+== Troubleshooting migrations
+You can perform the following troubleshooting tasks:
+
+* xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-viewing-migration-plan-resources_troubleshooting-3-4[Viewing migration plan resources by using the MTC web console]
+* xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-viewing-migration-plan-log_troubleshooting-3-4[Viewing the migration plan aggregated log file]
+* xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-using-mig-log-reader_troubleshooting-3-4[Using the migration log reader]
+* xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-accessing-performance-metrics_troubleshooting-3-4[Accessing performance metrics]
+* xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-using-must-gather_troubleshooting-3-4[Using the `must-gather` tool]
+* xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-debugging-velero-resources_troubleshooting-3-4[Using the Velero CLI to debug Backup and Restore CRs]
```

Comment:
Backup and Restore should be marked with backticks because they are CR names.

Reply:
Addressed